Snippets/ASB/ASB_8/AddressingStrategies/NamespacePartitioningExtension.cs

```diff
@@ -0,0 +1,15 @@
+using NServiceBus;
+using NServiceBus.Configuration.AdvanceExtensibility;
+
+#region custom-namespace-partitioning-strategy-extension
+
+public static class AzureServiceBusNamespacePartitioningStrategyExtensions
+{
+    public static AzureServiceBusNamespacePartitioningSettings MySetting(this AzureServiceBusNamespacePartitioningSettings extensionPoint)
```

Comment:
And V8 cannot be bad
![image](https://user-images.githubusercontent.com/1309622/27207351-747ce2d2-51fb-11e7-83dc-235a5958b994.png)